app/src/main/java/org/wildstang/wildrank/androidv2/activities/NotesActivity.java

```diff
@@ -0,0 +1,154 @@
+package org.wildstang.wildrank.androidv2.activities;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.ActionBarActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.widget.TextView;
+
+import com.couchbase.lite.CouchbaseLiteException;
+
+import org.wildstang.wildrank.androidv2.R;
+import org.wildstang.wildrank.androidv2.Utilities;
+import org.wildstang.wildrank.androidv2.data.DatabaseManager;
+import org.wildstang.wildrank.androidv2.fragments.NotesSixFragment;
+
+import java.io.IOException;
+
+/**
+ * Created by Liam on 2/21/2015.
+ */
+public class NotesActivity extends ActionBarActivity
+{
+    private Toolbar toolbar;
+    private String teamKeys;
+    private String matchKey;
+    private Boolean sixMode = true;
+    private String[] teams;
+    private String[] notes;
+
+    private NotesSixFragment sixFrag;
+
+    public static Intent createIntent(Context context, String matchKey, String teamKeys, String[] teams)
+    {
+        Intent i = new Intent(context, NotesActivity.class);
+        i.putExtra("team_keys", teamKeys);
```

Comment:
Declare the keys as private static class members just so we have an easy way to reference them.